plugins/Live/templates/getLastVisitsStart.twig

```diff
@@ -104,10 +104,10 @@
                                 {% endif %}
                             </span>
                         {% else %}
-                            {% set col = col + 1 %}
-                            {% if col >= 9 %}
-                                {% set col = 0 %}
+                            {% if folders is not empty %}
+                                {% set col = attribute(folders, action.url) %}
```

Comment:
Is it needed to check whether `folders[action.url]` is defined? and whether `action.url` is defined? I'm not really sure but would be save as something like this could happen I reckon. At least for `action.url` might not be defined